src/client/conn/http2.rs

```diff
@@ -132,23 +132,10 @@ where
     pub fn send_request(
         &mut self,
         req: Request<B>,
-    ) -> impl Future<Output = crate::Result<Response<IncomingBody>>> {
-        let sent = self.dispatch.send(req);
```

Comment:
What this was just:

```rust
let sent = self.dispatch.send(req).map_err(|_| ());
```

Would that be enough so that it doesn't need a named type?